modules/headless_ui/src/Redirect.php

```diff
@@ -11,10 +11,11 @@ class Redirect {
   public static function entityForm(array &$form, FormStateInterface $form_state) {
     /** @var \Drupal\Core\Entity\EntityFormInterface $form_object */
     $form_object = $form_state->getFormObject();
+    $form_id = is_null($form_object->getBaseFormId()) ? $form_object->getFormId() : $form_object->getBaseFormId();
```

Comment:
I'd request only a minor change here -- can we use the ?: syntax? So this would look like:

```php
$form_id = $form_object->getBaseFormId() ?: $form_object->getFormId();
```

Reply:
Done